Standards/CCSMB-15.md

```diff
@@ -0,0 +1,60 @@
+# _CCSMB-15_: Wide Printer Format and Wide Printer Format Array
+
+_Author: znepb_  
+_Version_: 1.0.0  
+_Last revised:_  
+The words MUST, MUST NOT, SHOULD, SHOULD NOT, and MAY are defined in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119).
+
+## Rational
+
+Wide Printer Format (2dj) is a file format used by [SwitchCraft peripherals](https://github.com/SwitchCraftCC/sc-peripherals) to print poster files. These files contain information to print onto these posters, each pixel being a different color similar to map files. Wide Format Print (A) (2dja), is similar to 2dj.
+
+## 2dj
+
+### Format
+
+| Information |                     |
+| ----------- | ------------------- |
+| Version     | 1.0.0               |
+| Type        | Wide Printer Format |
+| MIME        | `image/2dj`         |
+| Extensions  | `.2dj`              |
+
+Wide Printer Format files MUST follow [RFC 8259](https://datatracker.ietf.org/doc/html/rfc8259), "The JavaScript Object Notation (JSON) Data Interchange Format." These files MUST also use the file extension `.2dj`.
+
+### Keys
```

Comment:
Include something like "the file format should consist of a JSON file of the following keys", since it's not clear what you mean by immediately introducing a "keys" section followed by a list.

Reply:
Should be resolved in https://github.com/CCSMB/Standards/pull/31/commits/6ea0cd976ef0a1cde4d2062cbbaf7477ff3a8e25